model/yeast-GEM.txt

```diff
@@ -835,8 +835,8 @@ r_1081	4 H+[c] + uroporphyrinogen III[c] => 4 carbon dioxide[c] + coproporphyrin
 r_1082	2 S-adenosyl-L-methionine[c] + uroporphyrinogen III[c] => H+[c] + precorrin-2[c] + 2 S-adenosyl-L-homocysteine[c] 	YKR069W	  0.00	1000.00	  0.00
 r_1083	preuroporphyrinogen[c] => H2O[c] + uroporphyrinogen III[c] 	YOR278W	  0.00	1000.00	  0.00
 r_1084	D-glucose 1-phosphate[c] + H+[c] + UTP[c] <=> diphosphate[c] + UDP-D-glucose[c] 	YHL012W or YKL035W	-1000.00	1000.00	  0.00
-r_1085	ATP[c] + H+[c] + H2O[c] => ADP[c] + 2 H+[g] + phosphate[c] 	YBR127C and YDL185W and YEL027W and YEL051W and YGR020C and YHR026W and YHR039C-A and YKL080W and YLR447C and YMR054W and YOR332W and YPL234C and YPR036W	  0.00	1000.00	  0.00
```

Comment:
This reaction and the following `r_1086` is a little bit wired. @edkerk Just the H+ is changing to a different comp. I can understand this is maybe to represent the ATP conversion in golgi comp and vacuole comp. Maybe this can be a issue to solve later?

Reply:
It's an active proton pump. Seems biologically correct, although it does not have much influence in the model as there is also a free H+ diffusion reaction (`r_1826`). In general, ensuring strict H+ balancing could be the topic for future curation, although it is not necessarily straightforward, as this would tightly link H+ consumption and production, even while pH buffering in reality prevents this tight link.

Reply:
Yeah. I see your point. These reactions are fine. I misunderstand the role for these complexes. Sorry for that.